pkg/kubeapiserver/admission/initializer.go

```diff
@@ -67,6 +75,14 @@ func (i pluginInitializer) Initialize(plugin admission.Interface) {
 		wants.SetInternalClientSet(i.internalClient)
 	}
 
+	if wants, ok := plugin.(WantsToRun); ok {
```

Comment:
> If I write an admission controller that implements WantsToRun would I expect the Run to be the last operation invoked?

This is a good point.  Given the way we're likely to compose downstream in openshift, perhaps this should be promoted to a normal bit of `pkg/admission` instead of being an initializer.  Then an API server is responsible for creating, initializing, and then calling `Run` on its admission plugins.